sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EditEventSteps.java

```diff
@@ -1380,29 +1373,13 @@ public EditEventSteps(
         "^I click on New Sample and discard changes is asked$",
         () -> {
           webDriverHelpers.clickOnWebElementBySelector(NEW_SAMPLE_BUTTON);
-          if (webDriverHelpers.isElementVisibleWithTimeout(POPUP_DISCARD_CHANGES_BUTTON, 30)) {
```

Comment:
why removed?

Reply:
bug was fixed or disappeared the constant discard changes popup is not appearing any more thats why